content/roles_and_responsibilities.md

```diff
@@ -1,19 +1,64 @@
+    - On a Chapter-Verse Reference Question (CVR) or Chapter-Only Reference Question (CR), if the quizzer has given all the information in both the question and the answer and the complete question was not read, the quizmaster will ask, "What is your question?"
+    - On a Finish-The-Verse (FTV), Quote, or Finish This (FT) type question, once the quizzer has quoted to the end of the required verse or verses, if the quote is not yet word-perfect, the quizmaster will say, "Again."  This will occur as often as needed until the quizzer quotes the verse perfectly or the 30-second time limit expires.  
+        - The quizmaster is not allowed to say “again” until the quizzer has come to the end of the verse, each time through, as this would give the quizzer information about where their mistake was made.
+    - Once all of the information in both the question and answer have been given, but something needs to be fixed (e.g. Unique word) the quizmaster will say “Again”.
+    - On a Situation (SIT) type question, when the quizzer has completed the Quotation the quizmaster will say, “Quote is complete.”
+    - On an INT or MA Question, if all of the information in the answer has been given, but the quizzer is missing information in the question, the quizmaster will say “More”.
+    - On a Situation question, because the quotation is part of the question, the quizmaster will say “More” when required information from the quotation is missing.
+    - On a Q, Q2, CVR, MACVR, CR, MACR question, if either the chapter or verse number were not completely finished when the quizzer jumped, the quizmaster will say “More”. 
+        - Note: On a CVR/MACVR, CR/MACR Question once the quizzer has provided all the information in both the question and answer the quizmaster will say “What is your question”, if the response to the question supplied by the quizzer is “More”, the quizzer should consider whether the quizmaster had completely pronounced the chapter or verse number. 
+    - On a Reference type question, if the quizzer gives a question that is different than the one on the card, the quizmaster and answer judge must ensure that the question given is the exact same type as the one on the card.
+    - Consult with the answer judge(s) at any time he or she feels is necessary, provided it does not infringe upon the 30 seconds allotted the quizzer.
+    - If an explanation of a ruling is requested by a captain, the quizmaster will ask if any team wants to challenge before the explanation is given.
 
 ### Answer Judges
+- All conferring among quizmaster and answer judge(s) shall be done privately.
+    - It is the answer judge who will determine the accuracy of its content and make a ruling accordingly.
```

Comment:
I moved a lot of things around, and did not bring over a lot of AJ stuff.....

"Answer judge will check the 2011 NIV scripture when asked to verify the accuracy of the written answer.

All questions are checked for accuracy, construction, and distribution by the head answer judge prior to the quiz.

The answer judge may call a foul or rule infraction related to jumping."

The fouls bit is defined under Fouls. And the rulebook doesn't list calling fouls under QMs, so it's a tad inconsistent right now.

Reply:
Agreed.